app/assets/stylesheets/_results.scss

```diff
@@ -47,6 +47,12 @@
     }
   }
 
+  .result-uniform_title {
```

Comment:
One day we will discuss this at length. I will lose, but I want to say my words about it :)